nuxt.config.ts

```diff
@@ -18,21 +19,29 @@ export default defineNuxtConfig({
   typescript: {
     shim: false,
   },
-  ssr: false,
+  ssr: true,
   runtimeConfig: {
     secret: "", // override using NUXT_SECRET in .env
-    github_client_id: "", // same here
-    github_client_secret: "", // same here
     database_url: "", // same here
-    discord_client_secret: "",
-    discord_client_id: "",
+    discord_id: "",
```

Comment:
why has this changed?

Reply:
good question, do we have an answer to this?